frontend/src/components/Shared/Navbar/Navbar.jsx

```
@@ -125,22 +152,108 @@ const NavigationBar = () => {
             
             {/* Mobile menu button */}
-            <button className="md:hidden text-gray-300 hover:text-white focus:outline-none">
-              <svg className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M4 6h16M4 12h16m-7 6h7"></path>
-              </svg>
+            <button 
+              data-mobile-toggle
+              onClick={toggleMobileMenu}
+              className="md:hidden text-gray-300 hover:text-white focus:outline-none"
+              aria-label="Toggle menu"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)